core/optaplanner-constraint-streams-common/src/test/java/org/optaplanner/constraint/streams/common/inliner/HardSoftScoreInlinerTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates.
```

Comment:
@triceo (nitpick) Please check your IDE settings.
It's making PR's bigger than they need to be, increasing the potential merge conflict surface, making them longer to read.

Reply:
This is intentional, I have it configured to do that. As far as I know, this is what we want. If not, let's make sure everyone knows that.

Reply:
+1 to all do it the same way.

But probably no matter: I think this discussion is going away: Quarkus doesn't have these statements and Richard Fontana stated they serve no purpose. Let's discuss that first: https://issues.redhat.com/browse/PLANNER-2725 - and if we reject that, than we can make sure to sync.